.github/workflows/release.yml

```diff
@@ -0,0 +1,19 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+    - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+name: Upload Release Asset
+
+jobs:
+  build:
+    name: Upload Release Asset
+    runs-on: windows-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Build project # This would actually build your project, using zip for an example artifact
+        run: |
+          Compress-Archive -WhatIf -Path D:\a\ComposeUI\ComposeUI\src\shell\dotnet\Shell\bin\Debug\net6.0-windows\* composeui_win32.zip   
```

Comment:
Don't assume the workspace directory, use `${{ github.workspace }}/src/...` instead. In general, I would avoid using `\` as path separator since it's only supported on Windows, whereas `/` works on any platform.